performance/SwiftUIPerformanceQuickstart/PerformanceExample/Shared/Process.swift

```diff
@@ -217,5 +217,5 @@ enum ProcessStatus {
 enum ProcessAction: String, CaseIterable {
   case download = "Download"
   case classify = "Classify"
-  case upload = "Upload"
+//  case upload = "Upload"
```

Comment:
Why is this commented?

Reply:
The upload functionality will be added as a follow-up PR, and only download and classify are tested with the UI tests.

Reply:
maybe add a TODO comment?